about.html

```diff
@@ -4,61 +4,108 @@
 <head>
   <meta charset="utf-8">
 
-  <title>Sprint Challenge - About</title>
+  <title>Sprint Challenge | About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
+  <div class="container">
+      <!-- Navagation -->
+
+      <div class="main-nav">
+        <a class="logo" href="index.html"><img src="/img/lambda-black.png" alt=""></a>
```

Comment:
You don't want to put the slash before img. What that will do is start looking for the file in the root folder on your computer (/ on mac/linux, c:/ on Windows). If you take that out, everything should be good!